### PR TITLE
Implementation

### PR DESCRIPTION
**Pull request by outcatcher**
Implement bash script that can be used to set up infrastructure and export env vars as `eval $(./env.sh)`

### CODE REVIEW

File: env.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export OS_NETWORK_ID=$OS_NETWORK_ID
+export OS_VPC_ID=$OS_VPC_ID
+export OS_SUBNET_ID=$OS_SUBNET_ID
+export OS_IMAGE_ID=$OS_IMAGE_ID
+
+export TF_ACC=1
+export OS_POOL_NAME=ext_admin_net
+export OS_FLAVOR_NAME=s2.large.2
+export OS_AVAILABILITY_ZONE=eu-de-03
+export OS_EXTGW_ID=1
```
Comment (lego963) [inline env.sh]: this value doesn't work

File: env.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+OS_VPC_ID=$(get_var vpc_id)
+OS_SUBNET_ID=$(get_var subnet_id)
+OS_IMAGE_ID=$(get_var image_id)
+
+echo "
+# To get environment variables for running terraform accepance tests, run
+# eval \$(./env.sh)
+#
+# Please note that you still have to provide auth variables:
+# OS_AUTH_URL, OS_TENANT_NAME, OS_USERNAME, OS_PASSWORD, OS_USER_DOMAIN_NAME, OS_ACCESS_KEY, OS_SECRET_KEY
```
Comment (lego963) [inline env.sh:26]: What about these?
```
OS_PROJECT_NAME
OS_PROJECT_DOMAIN_NAME
```

File: env.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+export OS_IMAGE_ID=$OS_IMAGE_ID
+
+export TF_ACC=1
+export OS_POOL_NAME=ext_admin_net
+export OS_FLAVOR_NAME=s2.large.2
+export OS_AVAILABILITY_ZONE=eu-de-03
+export OS_EXTGW_ID=1
+export OS_FLAVOR_ID=s2.large.2
+export OS_DCS_ENVIRONMENT=1
+
```
Comment (lego963) [inline env.sh:42]: you missed OS_KEYPAIR_NAME

**Review by lego963 — CHANGES_REQUESTED** [review]
comments

File: env.sh (hunk 2 above)
Comment (outcatcher) [reply env.sh:26]: Please check `provider_test.go` for used variables

File: env.sh (hunk 1 above)
Comment (outcatcher) [reply env.sh]: Which one?

File: env.sh (hunk 2 above)
Comment (lego963) [reply env.sh:26]: ```OS_PROJECT_NAME``` exists

Comment (outcatcher) [reply env.sh:26]: `OS_TENANT_NAME` is enough:
image

File: env.sh (hunk 1 above)
Comment (lego963) [reply env.sh]: OS_EXTGW_ID requires ID, but `1` doens't do anything

File: resources.tf [hunk 4]
```diff
@@ -0,0 +1,39 @@
+data opentelekomcloud_images_image_v2 debian {
+  name = "Standard_Debian_10_latest"
+}
+
+resource opentelekomcloud_vpc_v1 vpc {
```
Comment (lego963) [inline resources.tf:9]: what about sec groups?

File: env.sh (hunk 1 above)
Comment (outcatcher) [reply env.sh]: What should be in `OS_EXTGW_ID`? It's used only a few tests but required if `TF_ACC` is set. Basically, I've never used this var.

File: resources.tf (hunk 4 above)
Comment (outcatcher) [reply resources.tf:9]: Are they required for terraform acceptance tests?

**Review by lego963 — APPROVED** [review]

**Pull request merged**